doc/040_backup.rst

```diff
@@ -40,6 +40,10 @@ As you can see, restic created a backup of the directory and was pretty
 fast! The specific snapshot just created is identified by a sequence of
 hexadecimal characters, ``40dc1520`` in this case.
 
+.. note:: When restic encounters an error during backup (e.g unreadable file, permission
```

Comment:
Does this paragraph belong into this PR? It looks like copy&paste from #112.

Reply:
Not sure what happened with that commit.  Probably doesn't belong in this PR.  Let me rebase it out.